src/pudl/package_data/phmsagas/column_maps/yearly_transmission_gathering_pipe_miles_by_class_location.csv

```diff
@@ -60,7 +60,3 @@ total_pipe_class_2_miles,,,,,,,,,,,,b4total_2,b4total_2,b4total_2,b4total_2,b4to
 total_pipe_class_3_miles,,,,,,,,,,,,b4total_3,b4total_3,b4total_3,b4total_3,b4total_3,b4total_3,b4total_3,b4total_3,b4total_3,partlc3tot,partlc3tot,partlc3tot,partlc3tot,partlc3tot,partlc3tot,partlc3tot,partlc3tot,partlc3tot,partlc3tot,partlc3tot,partlc3tot,partlc3tot
 total_pipe_class_4_miles,,,,,,,,,,,,b4total_4,b4total_4,b4total_4,b4total_4,b4total_4,b4total_4,b4total_4,b4total_4,b4total_4,partlc4tot,partlc4tot,partlc4tot,partlc4tot,partlc4tot,partlc4tot,partlc4tot,partlc4tot,partlc4tot,partlc4tot,partlc4tot,partlc4tot,partlc4tot
 total_pipe_class_total_miles,,,,,,,,,,,,b4total_t,b4total_t,b4total_t,b4total_t,b4total_t,b4total_t,b4total_t,b4total_t,b4total_t,partltotal,partltotal,partltotal,partltotal,partltotal,partltotal,partltotal,partltotal,partltotal,partltotal,partltotal,partltotal,partltotal
-total_pipe_hca_miles,,,,,,,,,,,,,,,,,,,,,partltrhcatotal,partltrhcatotal,partltrhcatotal,partltrhcatotal,partltrhcatotal,partltrhcatotal,partltrhcatotal,partltrhcatotal,partltrhcatotal,partltrhcatotal,partltrhcatotal,partltrhcatotal,partltrhcatotal
-total_pipe_192_710_miles,,,,,,,,,,,,,,,,,,,,,partltr192milestotal,partltr192milestotal,partltr192milestotal,partltr192milestotal,partltr192milestotal,partltr192milestotal,partltr192milestotal,partltr192milestotal,partltr192milestotal,partltr192milestotal,partltr192milestotal,partltr192milestotal,partltr192milestotal
-total_pipe_class_3_or_4_only_miles,,,,,,,,,,,,,,,,,,,,,partltrclass34total,partltrclass34total,partltrclass34total,partltrclass34total,partltrclass34total,partltrclass34total,partltrclass34total,partltrclass34total,partltrclass34total,partltrclass34total,partltrclass34total,partltrclass34total,partltrclass34total
-total_pipe_class_1_or_2_only_miles,,,,,,,,,,,,,,,,,,,,,partltrclass12total,partltrclass12total,partltrclass12total,partltrclass12total,partltrclass12total,partltrclass12total,partltrclass12total,partltrclass12total,partltrclass12total,partltrclass12total,partltrclass12total,partltrclass12total,partltrclass12total
```

Comment:
why are these guys gone? did they get double mapped or something?

Reply:
Yes, they got double mapped. See `transmission_pipe_hca_miles`, e.g. There aren't gathering miles in this section so it's only transmission, we decided for now to keep the totals closer to the source (and we could fill in the `total_` when we're cleaning, if we want).

Reply:
ah okay great!!